client/src/components/History/HistoryViews/JobHistoryModal.jsx

```diff
@@ -156,7 +188,7 @@ function JobHistoryContent({ jobDetails, jobDisplay, jobName, toggle }) {
             ),
```

Comment:
Should we hide `View in Data` files in v2?  will the mapping of storage systems be the same?   and if so, then should ensure its enabled (currently disabled as our last status was the initial "processing" one (see earlier comment).

Reply:
I've hidden it in v2!